.mailmap

```diff
@@ -33,3 +33,4 @@ Peiqi Shi <uestc.shi@gmail.com>
 Sergey Generalov <sergey@genbit.ru>
 Thomas Graf <thomas@cilium.io>
 Zhu Yan <hackzhuyan@gmail.com>
+Sean Winn <sean@isovalent.com> <seanmwinn@hotmail.com>
```

Comment:
So THAT's where this file is, ha! (Now I should remember for updating these in the future :-) )